### DISCUSSION TITLE
Termux(Android) support just needs a way to launch a browser for OAuth

### DISCUSSION THREAD

**Discussion by svgPhoenix** [ThioJoe/YT-Spammer-Purge#763] [Ideas]
I managed to get everything set up in Termux just by following the manual Linux installation instructions but, as I feared, the script failed to open a browser window to complete OAuth once I got a client_secrets file imported.

**Comment by Firecul** [comment]
I tend to ssh into my PC and run that if I need to from my mobile, it works well since I've normally used it that day and got all the tokens it needs already.

**Reply by svgPhoenix** [reply]
> I tend to ssh into my PC and run that if I need to from my mobile, it works well since I've normally used it that day and got all the tokens it needs already.

I don't have a PC to ssh into.

**Reply by Firecul** [reply]
That *would* be a flaw in my plan then.
